src/script.js

```diff
@@ -3,25 +3,46 @@ import googleAnalytics from './analytics'
 import syncLibrary from './sync-library'
 
 // Replace layerStyles and textLayerStyles in the document with selected theme library
-export default function() {
-  const libraries = sketch.getLibraries().filter(l => l.valid && l.enabled)
-  const [, selectionIndex, ok] = sketch.UI.getSelectionFromUser(
-    "Select a theme library to replace magically document's layer styles and symbols 🎉",
-    libraries.map(l => l.name)
-  )
-  googleAnalytics('Open Camilo', 'Alert', 'UI')
-
-  // Depending selected control, current document will sync with predefined brand
-  if (ok) {
-    const doc = sketch.getSelectedDocument()
-    const chosenLibrary = libraries[selectionIndex]
-
-    googleAnalytics('Camilo replacement with', chosenLibrary.name, 'Library')
-
-    syncLibrary(doc, chosenLibrary)
+export function getOptionSelected(libraries) {
+  
+  const options = [];
+  var optionSelected = NSComboBox.alloc().initWithFrame(NSMakeRect(0, 0, 200, 25));
```

Comment:
why would you go back to using NSComboBox and COSAlertWindow?

Reply:
I want to add a pair of radio buttons in the next release for "apply to whole document" and "apply only to selection". Didn't find a way to do this with API, looked at the API documentation. If there is a way, I will be happy to change it.

Reply:
I see